### PR TITLE
Add Wagon to closed source section

### PR DESCRIPTION
**Pull request by jweinstein**

### CODE REVIEW

**Comment by shinnn** [comment]
https://github.com/sindresorhus/awesome-electron/blob/d09b6b9c663355b9f4c68cfc0ba741387f04bfcd/contributing.md

**Comment by sindresorhus** [comment]
Also don't hard-wrap.

**Comment by arthurvr** [comment]
ping @jweinstein

**Comment by jweinstein** [comment]
wagon_uses_electron

**Comment by jweinstein** [comment]
@arthurvr updated, thanks!

File: readme.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ Some good apps written with Electron.
 - [Stremio](http://www.strem.io) - Next generation media center.
 - [RESTer](http://getrester.com) - Easy and beautiful way to test your APIs and servers.
 - [Typetalk](http://www.typetalk.in) - Share and discuss ideas with your team through instant messaging.
+- [Wagon](http://www.wagonhq.com) - A modern SQL editor: a better way to write queries, visualize results, and share data & charts.
```
Comment (shinnn) [inline readme.md]: `http` → `https`

Should use `HTTPS` as possible. http://mashable.com/2011/05/31/https-web-security/

**Comment by jweinstein** [comment]
@shinnn thanks! updated.

**Pull request closed without merging**